### ISSUE TITLE
Abnormal perception results when running a modified simple-inference.ipynb

### ISSUE THREAD

**Issue by HanFa** [traveller59/second.pytorch#249]
Greetings, everyone!

I am trying to use a pretrained model `car_onestage/voxelnet-27855.tckpt` and with config file `car.fhd.onestage.config`. I would like to pass in only one frame of LiDAR trace of Nuscenes at a time. 

My code is adapted from simple-inference.ipynb with very little modification, but it turns out giving abnormal detection results.

What did I change?
1.

I have commented out

```
#config_tool.change_detection_range(model_cfg, [-50, -50, 50, 50])
```

2. Instead of doing a pointcloud read from info files
```
info = infos[564]
v_path = info["point_cloud"]['velodyne_path']
v_path = str(root_path / v_path)
points = np.fromfile(
    v_path, dtype=np.float32, count=-1).reshape([-1, 4])
```

I just directory passed in the binary path,
```
    v_path = 'dataset/n008-2018-05-21-11-06-59-0400__LIDAR_TOP__1526915252897831.pcd.bin'
    points = np.fromfile(
        v_path, dtype=np.float32, count=-1).reshape([-1, 5])
    points = points[:, :4]
```

I reshaped the array with 5 because the Nuscenes LiDAR binary contains 5 fields `['x', 'y', 'z', 'intensity', 'ring']`

After running the scripts, the output map looks like (which is not quite reasonable)

![map](https://user-images.githubusercontent.com/20946893/61680210-c0d9f880-acd6-11e9-8f44-4aa628b33203.png)

Could anyone figure out any step I am missing since I have got stuck for hours? Any response will be highly appreciated. Thank you!

My script is attached:

```
#!/usr/bin/env python3

import numpy as np
import matplotlib.pyplot as plt
import pickle
from pathlib import Path

import torch
from google.protobuf import text_format
from second.utils import simplevis
from second.pytorch.train import build_network
from second.protos import pipeline_pb2
from second.utils import config_tool


if __name__ == '__main__':
    config_path = "second/configs/car.fhd.onestage.config"

    config = pipeline_pb2.TrainEvalPipelineConfig()
    with open(config_path, "r") as f:
        proto_str = f.read()
        text_format.Merge(proto_str, config)
    input_cfg = config.eval_input_reader
    model_cfg = config.model.second
    # config_tool.change_detection_range(model_cfg, [-50, -50, 50, 50])
    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")


    ckpt_path = "models/pretrained_models_v1.5/car_onestage/voxelnet-27855.tckpt"
    net = build_network(model_cfg).to(device).eval()
    net.load_state_dict(torch.load(ckpt_path))
    target_assigner = net.target_assigner
    voxel_generator = net.voxel_generator


    grid_size = voxel_generator.grid_size
    feature_map_size = grid_size[:2] // config_tool.get_downsample_factor(model_cfg)
    feature_map_size = [*feature_map_size, 1][::-1]


    #Here is the thing I changed
    v_path = '/home/fang/Desktop/nuscenes_subset/n008-2018-05-21-11-06-59-0400__LIDAR_TOP__1526915248396167.pcd.bin'
    anchors = target_assigner.generate_anchors(feature_map_size)["anchors"]
    anchors = torch.tensor(anchors, dtype=torch.float32, device=device)
    anchors = anchors.view(1, -1, 7)

    points = np.fromfile(
        v_path, dtype=np.float32, count=-1).reshape([-1, 5])


    points = points[:, :4]
    dic = voxel_generator.generate(points)
    voxels, coords, num_points = dic['voxels'], dic['coordinates'], dic['num_points_per_voxel']

    print(voxels.shape)
    # add batch idx to coords
    coords = np.pad(coords, ((0, 0), (1, 0)), mode='constant', constant_values=0)
    voxels = torch.tensor(voxels, dtype=torch.float32, device=device)
    coords = torch.tensor(coords, dtype=torch.int32, device=device)
    num_points = torch.tensor(num_points, dtype=torch.int32, device=device)


    example = {
        "anchors": anchors,
        "voxels": voxels,
        "num_points": num_points,
        "coordinates": coords,
    }
    pred = net(example)[0]

    print(pred)

    boxes_lidar = pred["box3d_lidar"].detach().cpu().numpy()
    vis_voxel_size = [0.1, 0.1, 0.1]
    vis_point_range = [-50, -30, -3, 50, 30, 1]
    bev_map = simplevis.point_to_vis_bev(points, vis_voxel_size, vis_point_range)
    bev_map = simplevis.draw_box_in_bev(bev_map, vis_point_range, boxes_lidar, [0, 255, 0], 2)

    plt.imshow(bev_map)
    plt.savefig('map.png')


```

**Comment by cm56uw** [comment]
It might be the problem mentioned in #201 , try to divided the intensities by 255

**Comment by HanFa** [comment]
@cm56uw thanks so much for helping me out!

After normalizing the intensities for Nuscenes data binary,
```
points[:, 3] = points[:, 3] / 255.0
```

it works much better.

![map](https://user-images.githubusercontent.com/20946893/61724862-c5d19300-ad3c-11e9-8e6c-4ecfeeebd287.png)
